### PR TITLE
20w12a update

### PR DESCRIPTION
**Pull request by Yuuki2012**
Updated to 20w12a, confirmed to work.

### CODE REVIEW

**Comment by Pyrofab** [comment]
Pretty sure this is not necessary, but thanks !

File: gradle.properties [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
+yarn_mappings=20w12a+build.5
+loader_version=0.7.8+build.188
 
 #Fabric api
-fabric_version=0.4.33+build.301-1.16
+fabric_version=0.5.5+build.311-1.16
 
 # Mod Properties
-mod_version = 1.3
+mod_version = 1.4
```
Comment (Pyrofab) [inline gradle.properties]:
```suggestion
mod_version = 1.3
```
No need to increment the mod version if nothing changes.

**Comment by Yuuki2012** [comment]
> Pretty sure this is not necessary, but thanks !

I don't know, it didn't start on my end using 20w09a, hence I changed it.

**Comment by Yuuki2012** [comment]
Had this error

```
net.minecraft.class_148: Rendering Block Entity
	at net.minecraft.class_824.method_3554(class_824.java:134) ~[intermediary-minecraft-20w12a-client.jar:?]
	at net.minecraft.class_824.method_3555(class_824.java:95) ~[intermediary-minecraft-20w12a-client.jar:?]
	at net.minecraft.class_761.method_22710(class_761.java:1184) ~[intermediary-minecraft-20w12a-client.jar:?]
	at net.minecraft.class_757.method_3188(class_757.java:718) ~[intermediary-minecraft-20w12a-client.jar:?]
	at net.minecraft.class_757.method_3192(class_757.java:543) ~[intermediary-minecraft-20w12a-client.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1003) ~[intermediary-minecraft-20w12a-client.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:631) [intermediary-minecraft-20w12a-client.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:204) [intermediary-minecraft-20w12a-client.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_241]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_241]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_241]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_241]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192) [fabric-loader-0.7.8+build.188.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:138) [fabric-loader-0.7.8+build.188.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26) [fabric-loader-0.7.8+build.188.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_241]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_241]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_241]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_241]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
Caused by: java.lang.NoSuchMethodError: net.minecraft.class_2680.method_11614()Lnet/minecraft/class_2248;
	at ladysnake.beem.BeeNestBlockEntityRenderer.render(BeeNestBlockEntityRenderer.java:23) ~[beem-1.3.jar:?]
	at ladysnake.beem.BeeNestBlockEntityRenderer.method_3569(BeeNestBlockEntityRenderer.java:13) ~[beem-1.3.jar:?]
	at net.minecraft.class_824.method_23079(class_824.java:106) ~[intermediary-minecraft-20w12a-client.jar:?]
	at net.minecraft.class_824.method_23081(class_824.java:95) ~[intermediary-minecraft-20w12a-client.jar:?]
	at net.minecraft.class_824.method_3554(class_824.java:127) ~[intermediary-minecraft-20w12a-client.jar:?]
	... 22 more
```

**Pull request merged**